src/components/KFileUpload/KFileUpload.cy.ts

```diff
@@ -86,7 +86,7 @@ describe('KFileUpload', () => {
       cy.wrap(Cypress.vueWrapper.emitted()).should('have.property', 'file-added')
       cy.wrap(Cypress.vueWrapper.emitted()).should('have.property', 'input')
     })
-    cy.getTestId('remove-button').should('exist').click().then(() => {
+    cy.getTestId('file-upload-button').should('exist').should('contain.text', 'Clear').click().then(() => {
```

Comment:
issue: you need to add another test that clicking on the button triggers the upload; it's currently broken.